crates/solidity/outputs/npm/crate/src/generated/napi/napi_ast_selectors.rs

```diff
@@ -0,0 +1,2958 @@
+// This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
```

Comment:
looks like this was added during rebase? it was already deleted earlier.

Reply:
done